docs/terminal/amberleyoakey.md

```diff
@@ -4,7 +4,103 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
 
+--8<-- "includes/abbreviations.md"
+
+## YAMB Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Amberley TCU   | Amberley Approach   | 126.200        | AM_APP                                   |
+
+
+
+## Airspace
+### AM TCU
+#### Restricted Areas
+By Default Amberley owns all of the R625 Restricted Areas, detailed below:
+
+- R625A (`A015`-`A085`)  
+- R625B (`A025`-`A085`)  
+- R625C (`A045`-`A085`)  
+- R625D (`A085`-`F210`)  
+
+### Optional, as required
+AM APP can negotiate further airspace releases from surrounding ENR sectors of the following Restricted Areas:
```

Comment:
Maybe just a note to indicate *why* a release may be negotiated (i.e. to facilitate planned military operations, not just for the hell of it).